tests/test_api.py

```diff
@@ -34,3 +34,6 @@ def tracer(self):
     def test_binary_propagation(self):
         # TODO binary codecs are not implemented at the moment
         pass
+
+    def is_parent(self, parent, span):
```

Comment:
Hi, I do not see this function is used any where (at least for now).

Reply:
This function is required by opentracing [here](https://github.com/opentracing/opentracing-python/blob/2ce13e2dd19e263daa5db88800cf6e84691a0b8e/opentracing/harness/api_check.py#L50).